ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -3917,3 +3917,32 @@ def test_tensorflow_Zeta(
         x=x[0],
         q=x[1],
     )
+
+# Imag
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.Imag",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=["complex64"],
+    ),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_Imag(
+    *,
+    dtype_and_x,
+    test_flags,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        test_flags=test_flags,
+        frontend=frontend,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        rtol=1e-2,
+        atol=1e-2,
```

Comment:
Are those necessary?

Reply:
Not really, the test runs perfectly fine without it but I was going through the test_math.py and the test for the standard imag had one included and from an engineering pov it kind of makes sense to declare some tolerance for extreme edge cases so I transposed the set from the test_math.py

Reply:
As for the first issue, I conferred with a Ivy Engineer and they suggested that I use the map_raw_ops_alias and given that the math.imag does not require or pass a Tout argument I chose to neglect it, if it is to be included do let me know

Reply:
We know do use safety factor instead, also I'm thinking this tolerance is really high. We will remove it from our other tests as well